cosmos/core/airflow.py

```diff
@@ -35,6 +35,12 @@ def get_airflow_task(task: Task, dag: DAG, task_group: TaskGroup | None = None)
     for k, v in task.airflow_task_config.items():
         task_kwargs[k] = v
 
+    # Set the on_warning_callback of source node in task_kwargs
```

Comment:
The comment mentions source node here, but it appears we set it in all nodes? Should we drop the word source from here in the comment?

Reply:
on_warning_callback get set only for node type source but I have removed it